lib/aytests/image_builder.rb

```diff
@@ -22,6 +22,7 @@ class ImageBuilder
     SSH_ADDRESS = "127.0.0.1"
     SSH_PORT = "22"
     WEBSERVER_PORT = "8888"
+    MAC_ADDRESS = "02:00:00:12:34:56"
```

Comment:
Because is a 'locally administered one' (the `02` means that). I plan to add a command line switch to specify one. More info here [in the wikipedia](https://en.wikipedia.org/wiki/MAC_address).